react/components/FiltersContainer.js

```diff
@@ -1,49 +1,155 @@
+/* global __RUNTIME__ */
+import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-import { Collapse } from 'react-collapse'
+import { flatten } from 'ramda'
 
-import ArrowDown from '../images/arrow-down.svg'
-import ArrowUp from '../images/arrow-up.svg'
+import SelectedFilters from './SelectedFilters'
+import AvailableFilters from './AvailableFilters'
+import AccordionFilterContainer from './AccordionFilterContainer'
+import { formatCategoriesTree, mountOptions, getMapByType } from '../constants/SearchHelpers'
+import { facetOptionShape, paramShape } from '../constants/propTypes'
+
+export const CATEGORIES_TYPE = 'Categories'
+export const BRANDS_TYPE = 'Brands'
+export const PRICE_RANGES_TYPE = 'PriceRanges'
+export const SPECIFICATION_FILTERS_TYPE = 'SpecificationFilters'
+
+const CATEGORIES_TITLE = 'search.filter.title.categories'
+const BRANDS_TITLE = 'search.filter.title.brands'
+const PRICE_RANGES_TITLE = 'search.filter.title.price-ranges'
 
 export default class FiltersContainer extends Component {
   static propTypes = {
-    filters: PropTypes.arrayOf(PropTypes.object).isRequired,
-    children: PropTypes.func.isRequired,
-    title: PropTypes.string.isRequired,
+    getLinkProps: PropTypes.func.isRequired,
```

Comment:
I think they're a lot of familiar props between these components, maybe is better extract to reuse